README.md

```diff
@@ -59,7 +59,8 @@ Browsers don't have native support for Grist [yet :-)] but you can make a little
   * You can also pass `initialData: 'path/to/data.csv'` to import a CSV file into a new table. In this case `initialFile` is optional.
   * There is also `initialContent` option. You can use it to pass the content of a CSV file.
   * You can also pass `elementId: 'element-id` to open Grist within an element in your page.
-    - If using `elementId`, and serving a `.grist` or `.csv` file at a URL that requires cookie-based authentication, be aware that Firefox may not support this yet ([bug report](https://bugzilla.mozilla.org/show_bug.cgi?id=1741489)).
+    - If using `elementId`, and serving a `.grist` or `.csv` file at a URL that requires cookie-based authentication, be aware that Firefox may not support this yet ([bug report](https://bugzilla.mozilla.org/show_bug.cgi?id=1741489)). It
```

Comment:
I'm comfortable just deleting this line now, since it applies specifically to srcdoc, and you've handled srcdoc